rollup.config.ts

```diff
@@ -124,7 +124,10 @@ function getPlugins({ format, mode }: Config): Plugin[] {
   if (mode === 'slim') {
```

Comment:
This is only running when `slim` is selected, I think we want to use `graphql-web-lite` in every case.